storage/buckets/add_bucket_label.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buckets
+
+// [START storage_add_bucket_label]
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"cloud.google.com/go/storage"
+)
+
+// addBucketLabel adds a label on a bucket.
+func addBucketLabel(w io.Writer, bucketName, labelName, labelValue string) error {
+	// bucketName := "bucket-name"
+	// labelName := "label-name"
+	// labelValue := "label-value"
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("storage.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	ctx, cancel := context.WithTimeout(ctx, time.Second*10)
+	defer cancel()
+
+	bucket := client.Bucket(bucketName)
+	bucketAttrsToUpdate := storage.BucketAttrsToUpdate{}
+	bucketAttrsToUpdate.SetLabel(labelName, labelValue)
```

Comment:
Good question, I've search for that opportunity, but it seems like we don't have it because labels in `BucketAttrsToUpdate` is [private](https://github.com/googleapis/google-cloud-go/blob/01ee6713322b0075df6dfad89b2031b6e4dbedaa/storage/bucket.go#L698). 
Another way to set [label](https://github.com/googleapis/google-cloud-go/blob/01ee6713322b0075df6dfad89b2031b6e4dbedaa/storage/bucket.go#L292) is to do it on `storage.BucketAttrs` level and to call `Bucket().Attrs()`, but this method is out of the general concept, I think. Also [example_test.go](https://github.com/googleapis/google-cloud-go/blob/01ee6713322b0075df6dfad89b2031b6e4dbedaa/storage/example_test.go#L133) uses `SetLabel`, that's why I decided to use it too